README.md

```diff
@@ -25,6 +25,43 @@ yarn add web3
 -   :gear: [NodeJS](https://nodejs.org/) (LTS/Fermium)
 -   :toolbox: [Yarn](https://yarnpkg.com/)/[Lerna](https://lerna.js.org/)
 
+## Web3 Tree shaking Support Guide
+
+1. Use the `production` mode configuration option to enable various optimizations including minification and tree shaking. Set to your webpack.config:
```

Comment:
Instead of duplicate , you may add link to 4.x treeshaking documentation guide in this readme here?

Reply:
I've added section `Useful links`